2022-09-05 15:02:13,612 INFO    MainThread:22900 [wandb_setup.py:_flush():71] setting env: {}
2022-09-05 15:02:13,612 WARNING MainThread:22900 [wandb_setup.py:_flush():71] could not find program at run.py
2022-09-05 15:02:13,612 INFO    MainThread:22900 [wandb_setup.py:_flush():71] setting login settings: {}
2022-09-05 15:02:13,612 INFO    MainThread:22900 [wandb_init.py:_log_setup():371] Logging user logs to /home/zzx/deepke/DeepKE/example/re/standard/logs/2022-09-05_15-02-12/wandb/run-20220905_150213-3tuqyotn/logs/debug.log
2022-09-05 15:02:13,612 INFO    MainThread:22900 [wandb_init.py:_log_setup():372] Logging internal logs to /home/zzx/deepke/DeepKE/example/re/standard/logs/2022-09-05_15-02-12/wandb/run-20220905_150213-3tuqyotn/logs/debug-internal.log
2022-09-05 15:02:13,612 INFO    MainThread:22900 [wandb_init.py:init():404] calling init triggers
2022-09-05 15:02:13,612 INFO    MainThread:22900 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-09-05 15:02:13,612 INFO    MainThread:22900 [wandb_init.py:init():449] starting backend
2022-09-05 15:02:13,612 INFO    MainThread:22900 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-09-05 15:02:13,624 INFO    MainThread:22900 [backend.py:ensure_launched():199] starting backend process...
2022-09-05 15:02:13,632 INFO    MainThread:22900 [backend.py:ensure_launched():204] started backend process with pid: 22952
2022-09-05 15:02:13,633 INFO    MainThread:22900 [wandb_init.py:init():458] backend started and connected
2022-09-05 15:02:13,634 INFO    MainThread:22900 [wandb_init.py:init():519] updated telemetry
2022-09-05 15:02:13,648 INFO    MainThread:22900 [wandb_init.py:init():549] communicating current version
2022-09-05 15:02:15,292 INFO    MainThread:22900 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.13.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-09-05 15:02:15,292 INFO    MainThread:22900 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-09-05 15:02:15,582 INFO    MainThread:22900 [wandb_init.py:init():592] starting run threads in backend
2022-09-05 15:02:17,194 INFO    MainThread:22900 [wandb_run.py:_console_start():1816] atexit reg
2022-09-05 15:02:17,194 INFO    MainThread:22900 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-09-05 15:02:17,194 INFO    MainThread:22900 [wandb_run.py:_redirect():1695] Redirecting console.
2022-09-05 15:02:17,196 INFO    MainThread:22900 [wandb_run.py:_redirect():1751] Redirects installed.
2022-09-05 15:02:17,196 INFO    MainThread:22900 [wandb_init.py:init():619] run started, returning control to user process
2022-09-05 15:02:23,061 INFO    MainThread:22900 [wandb_watch.py:watch():43] Watching
2022-09-05 15:02:26,375 INFO    MainThread:22900 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 255
2022-09-05 15:02:26,375 INFO    MainThread:22900 [wandb_run.py:_restore():1758] restore
